extensions/todo-list/src/menu_bar.tsx

```diff
@@ -0,0 +1,31 @@
+import { TodoItem, TodoSections, todoAtom } from "./atoms";
+
+import { MenuBarExtra } from "@raycast/api";
+import MenuBarTodoItem from "./menu_bar_todo_item";
+import { SECTIONS_DATA } from "./config";
+import { useAtom } from "jotai";
+
+export default function MenuBar() {
+  const [todoSections] = useAtom(todoAtom);
+
+  return (
+    <MenuBarExtra icon="../assets/command-icon-menubar.png" tooltip="Your Pull Requests">
```

Comment:
if you have no todos, `MenuBarExtra` will have no children, so it won't have a menu, so clicking it will appear to do nothing, which could be a bit confusing. To avoid that awkwardness, I'd suggest adding a placeholder Item when there are no todos.